src/modules/client/components/planning/EventEditionModal.vue

```diff
@@ -19,8 +19,7 @@
           <ni-datetime-range caption="Dates et heures de l'évènement" :value="editedEvent.dates" required-field
             :disable="isBilledIntervention" :error="validations.dates.$error" @input="update($event, 'dates')"
             @blur="validations.dates.$touch" :disable-start-date="isEventTimeStamped" :max="customerStoppedDate"
-            disable-end-date :disable-start-hour="!!editedEvent.startDateTimeStampedCount"
-            :disable-end-hour="!!editedEvent.endDateTimeStampedCount" />
+            disable-end-date :disable-start-hour="!!startDateTimeStamped" :disable-end-hour="!!endDateTimeStamped" />
         </template>
         <template v-if="editedEvent.type === INTERVENTION">
           <ni-select v-if="isCustomerPlanning" in-modal caption="Auxiliaire" :value="editedEvent.auxiliary"
```

Comment:
je sais pas pourquoi c'est pas le cas mais je pense qu'on peut bloquer ce champ aussi quand c'est timestamped

Reply:
`timeStamped = !!this.startDateTimeStamped || !!this.endDateTimeStamped;` donc si on bloque tout le time-range, on ne pourra plus modifier l'heure de début ou l'heure de fin si un l'event n'est que `startDateTimeStamped` ou `endDateTimeStamped`, non ?

Reply:
Je pense qu'Ulysse fait reference au champs auxiliaire lorsqu'on est sur le planning beneficiaire :) Tu as raison, je le change.